vm/devices/support/fs/lxutil/src/windows/mod.rs

```diff
@@ -5,54 +5,53 @@
 #![expect(unsafe_code)]
 #![allow(clippy::undocumented_unsafe_blocks)]
 
+#[macro_use]
```

Comment:
There's no real need to `macro_use` anymore: https://users.rust-lang.org/t/what-does-macro-use-mean/62431/3